build.js

```diff
@@ -70,12 +72,14 @@ const plugins = ({ activity, styles, cssRestrictions }) => {
 };
 
 // Loop through files and build each one
-const buildActivities = async (activitiesFilter = []) => {
+const buildActivities = async (activitiesFilter = [], activitiesGroup) => {
 	let activitiesToBuild = [];
 	if (activitiesFilter.length) {
-		activitiesToBuild = activitiesJSON.activities.filter(activity => activitiesFilter.includes(activity.activity));
+		console.log('hit?');
```

Comment:
Console logs like this might be the only way we can debug stuff on a day to day basis when we deploy things.. then dig around trying to find out what went sideways. maybe we could add something here like `console.log('Activities To Build: ', activitiesToBuild);`